[package] cairo/1.17.4: freetype-config not found during configure, when installed with host and build profile setted

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **cairo/1.17.4*
  * Operating System+version: **conanio/clang11*
  * Compiler+version: **clang 11**
  * Docker image: **conanio/clang11**
  * Conan version: **conan 1.35.2**
  * Python version: **Python 3.8.2**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Default settings
	os=Linux
	os_build=Linux
	arch=x86_64
	arch_build=x86_64
	compiler=clang
	compiler.version=11
	compiler.libcxx=libstdc++
	build_type=Release
```


### Steps to reproduce (Include if Applicable)
```
docker run --rm conanio/clang11 bash -c "conan install cairo/1.17.4@ --build missing -pr:h ~/.conan/profiles/default -pr:b ~/.conan/profiles/default -o cairo:with_xcb=False -o cairo:with_xlib=False -o cairo:with_xlib_xrender=False -o cairo:with_glib=True"
```

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
docker run --rm conanio/clang11 bash -c "conan install cairo/1.17.4@ --build missing -pr:h ~/.conan/profiles/default -pr:b ~/.conan/profiles/default -o cairo:with_xcb=False -o cairo:with_xlib=False -o cairo:with_xlib_xrender=False -o cairo:with_glib=True" 
Auto detecting your dev setup to initialize the default profile (/home/conan/.conan/profiles/default)
CC and CXX: clang, clang++ 
Found clang 11.1
clang>=8, using the major as version
Default settings
	os=Linux
	os_build=Linux
	arch=x86_64
	arch_build=x86_64
	compiler=clang
	compiler.version=11
	compiler.libcxx=libstdc++
	build_type=Release
*** You can change them in /home/conan/.conan/profiles/default ***
*** Or override with -s compiler='other' -s ...s***


Configuration (profile_host):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=clang
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
cairo:with_glib=True
cairo:with_xcb=False
cairo:with_xlib=False
cairo:with_xlib_xrender=False
[build_requires]
[env]

Configuration (profile_build):
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=clang
compiler.libcxx=libstdc++
compiler.version=11
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

cairo/1.17.4: Not found in local cache, looking in remotes...
cairo/1.17.4: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
cairo/1.17.4: Downloaded recipe revision 0
freetype/2.10.4: Not found in local cache, looking in remotes...
freetype/2.10.4: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
freetype/2.10.4: Downloaded recipe revision 0
libpng/1.6.37: Not found in local cache, looking in remotes...
libpng/1.6.37: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libpng/1.6.37: Downloaded recipe revision 0
zlib/1.2.11: Not found in local cache, looking in remotes...
zlib/1.2.11: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
zlib/1.2.11: Downloaded recipe revision 0
bzip2/1.0.8: Not found in local cache, looking in remotes...
bzip2/1.0.8: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
bzip2/1.0.8: Downloaded recipe revision 0
brotli/1.0.9: Not found in local cache, looking in remotes...
brotli/1.0.9: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
brotli/1.0.9: Downloaded recipe revision 0
fontconfig/2.13.93: Not found in local cache, looking in remotes...
fontconfig/2.13.93: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
fontconfig/2.13.93: Downloaded recipe revision 0
expat/2.3.0: Not found in local cache, looking in remotes...
expat/2.3.0: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
expat/2.3.0: Downloaded recipe revision 0
libuuid/1.0.3: Not found in local cache, looking in remotes...
libuuid/1.0.3: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libuuid/1.0.3: Downloaded recipe revision 0
glib/2.67.5: Not found in local cache, looking in remotes...
glib/2.67.5: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
glib/2.67.5: Downloaded recipe revision 0
libffi/3.3: Not found in local cache, looking in remotes...
libffi/3.3: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libffi/3.3: Downloaded recipe revision 0
pcre/8.44: Not found in local cache, looking in remotes...
pcre/8.44: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
pcre/8.44: Downloaded recipe revision 0
libelf/0.8.13: Not found in local cache, looking in remotes...
libelf/0.8.13: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libelf/0.8.13: Downloaded recipe revision 0
libmount/2.36: Not found in local cache, looking in remotes...
libmount/2.36: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libmount/2.36: Downloaded recipe revision 0
libselinux/3.1: Not found in local cache, looking in remotes...
libselinux/3.1: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libselinux/3.1: Downloaded recipe revision 0
pcre2/10.36: Not found in local cache, looking in remotes...
pcre2/10.36: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
pcre2/10.36: Downloaded recipe revision 0
pixman/0.40.0: Not found in local cache, looking in remotes...
pixman/0.40.0: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
pixman/0.40.0: Downloaded recipe revision 0
libtool/2.4.6: Not found in local cache, looking in remotes...
libtool/2.4.6: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libtool/2.4.6: Downloaded recipe revision 0
automake/1.16.3: Not found in local cache, looking in remotes...
automake/1.16.3: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
automake/1.16.3: Downloaded recipe revision 0
autoconf/2.69: Not found in local cache, looking in remotes...
autoconf/2.69: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
autoconf/2.69: Downloaded recipe revision 0
m4/1.4.18: Not found in local cache, looking in remotes...
m4/1.4.18: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
m4/1.4.18: Downloaded recipe revision 0
pkgconf/1.7.3: Not found in local cache, looking in remotes...
pkgconf/1.7.3: Trying with 'conan-center'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
pkgconf/1.7.3: Downloaded recipe revision 0
Installing package: cairo/1.17.4
Requirements
    brotli/1.0.9 from 'conan-center' - Downloaded
    bzip2/1.0.8 from 'conan-center' - Downloaded
    cairo/1.17.4 from 'conan-center' - Downloaded
    expat/2.3.0 from 'conan-center' - Downloaded
    fontconfig/2.13.93 from 'conan-center' - Downloaded
    freetype/2.10.4 from 'conan-center' - Downloaded
    glib/2.67.5 from 'conan-center' - Downloaded
    libelf/0.8.13 from 'conan-center' - Downloaded
    libffi/3.3 from 'conan-center' - Downloaded
    libmount/2.36 from 'conan-center' - Downloaded
    libpng/1.6.37 from 'conan-center' - Downloaded
    libselinux/3.1 from 'conan-center' - Downloaded
    libuuid/1.0.3 from 'conan-center' - Downloaded
    pcre/8.44 from 'conan-center' - Downloaded
    pcre2/10.36 from 'conan-center' - Downloaded
    pixman/0.40.0 from 'conan-center' - Downloaded
    zlib/1.2.11 from 'conan-center' - Downloaded
Packages
    brotli/1.0.9:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    bzip2/1.0.8:a0279858937b0968952a9784938616e26eb7c2dd - Download
    cairo/1.17.4:047d690cb0db4aa8f5feeff56c57a66b01987249 - Build
    expat/2.3.0:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    fontconfig/2.13.93:15af59b859610203277a1fbe9ab3cd592946e5b0 - Download
    freetype/2.10.4:59acdf248f8581cd189bde2c88b41e049eae7a62 - Download
    glib/2.67.5:95cb03f69b063acaf3e0248d172a4e0b702071f8 - Download
    libelf/0.8.13:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    libffi/3.3:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    libmount/2.36:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    libpng/1.6.37:4305977d9227c3156a3f9d8759c25afec76d9f0a - Download
    libselinux/3.1:8124df200a5b9d9925940d4c54fc8fa8d816cc79 - Download
    libuuid/1.0.3:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    pcre/8.44:0a0c659aef719d0748930c686eaa772f838c1520 - Download
    pcre2/10.36:c5cd7a5e52d42ff90834b8d0393b4067fc309680 - Download
    pixman/0.40.0:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
    zlib/1.2.11:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download
Build requirements
    autoconf/2.69 from 'conan-center' - Downloaded
    automake/1.16.3 from 'conan-center' - Downloaded
    libtool/2.4.6 from 'conan-center' - Downloaded
    m4/1.4.18 from 'conan-center' - Downloaded
    pkgconf/1.7.3 from 'conan-center' - Downloaded
Build requirements packages
    autoconf/2.69:cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31 - Download
    automake/1.16.3:258f6f05ca54813b3d7180161753df7f9ccf7e1d - Download
    libtool/2.4.6:e33671439e0af363817f093b37f44bf2e210d7b4 - Download
    m4/1.4.18:24647d9fe8ec489125dfbae4b3ebefaf7581674c - Download
    pkgconf/1.7.3:d988447fa516eac7400b2f34e2d4b89e42b4b1a8 - Download

Installing (downloading, building) binaries...
brotli/1.0.9: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
brotli/1.0.9: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
brotli/1.0.9: Downloaded package revision 0
bzip2/1.0.8: Retrieving package a0279858937b0968952a9784938616e26eb7c2dd from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
bzip2/1.0.8: Package installed a0279858937b0968952a9784938616e26eb7c2dd
bzip2/1.0.8: Downloaded package revision 0
expat/2.3.0: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
expat/2.3.0: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
expat/2.3.0: Downloaded package revision 0
libelf/0.8.13: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libelf/0.8.13: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
libelf/0.8.13: Downloaded package revision 0
libffi/3.3: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libffi/3.3: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
libffi/3.3: Downloaded package revision 0
libmount/2.36: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libmount/2.36: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
libmount/2.36: Downloaded package revision 0
libuuid/1.0.3: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libuuid/1.0.3: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
libuuid/1.0.3: Downloaded package revision 0
m4/1.4.18: Retrieving package 24647d9fe8ec489125dfbae4b3ebefaf7581674c from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
m4/1.4.18: Package installed 24647d9fe8ec489125dfbae4b3ebefaf7581674c
m4/1.4.18: Downloaded package revision 0
pixman/0.40.0: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
pixman/0.40.0: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
pixman/0.40.0: Downloaded package revision 0
pkgconf/1.7.3: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
pkgconf/1.7.3: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
pkgconf/1.7.3: Downloaded package revision 0
zlib/1.2.11: Retrieving package d988447fa516eac7400b2f34e2d4b89e42b4b1a8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
zlib/1.2.11: Package installed d988447fa516eac7400b2f34e2d4b89e42b4b1a8
zlib/1.2.11: Downloaded package revision 0
autoconf/2.69: Retrieving package cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
autoconf/2.69: Package installed cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31
autoconf/2.69: Downloaded package revision 0
libpng/1.6.37: Retrieving package 4305977d9227c3156a3f9d8759c25afec76d9f0a from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libpng/1.6.37: Package installed 4305977d9227c3156a3f9d8759c25afec76d9f0a
libpng/1.6.37: Downloaded package revision 0
pcre/8.44: Retrieving package 0a0c659aef719d0748930c686eaa772f838c1520 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
pcre/8.44: Package installed 0a0c659aef719d0748930c686eaa772f838c1520
pcre/8.44: Downloaded package revision 0
pcre2/10.36: Retrieving package c5cd7a5e52d42ff90834b8d0393b4067fc309680 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
pcre2/10.36: Package installed c5cd7a5e52d42ff90834b8d0393b4067fc309680
pcre2/10.36: Downloaded package revision 0
automake/1.16.3: Retrieving package 258f6f05ca54813b3d7180161753df7f9ccf7e1d from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
automake/1.16.3: Package installed 258f6f05ca54813b3d7180161753df7f9ccf7e1d
automake/1.16.3: Downloaded package revision 0
freetype/2.10.4: Retrieving package 59acdf248f8581cd189bde2c88b41e049eae7a62 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
freetype/2.10.4: Package installed 59acdf248f8581cd189bde2c88b41e049eae7a62
freetype/2.10.4: Downloaded package revision 0
libselinux/3.1: Retrieving package 8124df200a5b9d9925940d4c54fc8fa8d816cc79 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libselinux/3.1: Package installed 8124df200a5b9d9925940d4c54fc8fa8d816cc79
libselinux/3.1: Downloaded package revision 0
fontconfig/2.13.93: Retrieving package 15af59b859610203277a1fbe9ab3cd592946e5b0 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
fontconfig/2.13.93: Package installed 15af59b859610203277a1fbe9ab3cd592946e5b0
fontconfig/2.13.93: Downloaded package revision 0
glib/2.67.5: Retrieving package 95cb03f69b063acaf3e0248d172a4e0b702071f8 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
glib/2.67.5: Package installed 95cb03f69b063acaf3e0248d172a4e0b702071f8
glib/2.67.5: Downloaded package revision 0
libtool/2.4.6: Retrieving package e33671439e0af363817f093b37f44bf2e210d7b4 from remote 'conan-center' 
Downloading conanmanifest.txt
Downloading conaninfo.txt
Downloading conan_package.tgz
libtool/2.4.6: Package installed e33671439e0af363817f093b37f44bf2e210d7b4
libtool/2.4.6: Downloaded package revision 0
bzip2/1.0.8: Appending PATH environment variable: /home/conan/.conan/data/bzip2/1.0.8/_/_/package/a0279858937b0968952a9784938616e26eb7c2dd/bin
m4/1.4.18: Appending PATH environment variable: /home/conan/.conan/data/m4/1.4.18/_/_/package/24647d9fe8ec489125dfbae4b3ebefaf7581674c/bin
m4/1.4.18: Setting M4 environment variable: /home/conan/.conan/data/m4/1.4.18/_/_/package/24647d9fe8ec489125dfbae4b3ebefaf7581674c/bin/m4
pkgconf/1.7.3: Appending PATH env var: /home/conan/.conan/data/pkgconf/1.7.3/_/_/package/d988447fa516eac7400b2f34e2d4b89e42b4b1a8/bin
pkgconf/1.7.3: Setting PKG_CONFIG env var: /home/conan/.conan/data/pkgconf/1.7.3/_/_/package/d988447fa516eac7400b2f34e2d4b89e42b4b1a8/bin/pkgconf
pkgconf/1.7.3: Appending AUTOMAKE_CONAN_INCLUDES env var: /home/conan/.conan/data/pkgconf/1.7.3/_/_/package/d988447fa516eac7400b2f34e2d4b89e42b4b1a8/bin/aclocal
autoconf/2.69: Appending PATH env var with : /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin
autoconf/2.69: Setting AC_MACRODIR to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/share/autoconf
autoconf/2.69: Setting AUTOCONF to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autoconf
autoconf/2.69: Setting AUTORECONF to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autoreconf
autoconf/2.69: Setting AUTOHEADER to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autoheader
autoconf/2.69: Setting AUTOM4TE to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autom4te
autoconf/2.69: Setting AUTOM4TE_PERLLIBDIR to /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/share/autoconf
pcre/8.44: Appending PATH environment variable: /home/conan/.conan/data/pcre/8.44/_/_/package/0a0c659aef719d0748930c686eaa772f838c1520/bin
pcre2/10.36: Appending PATH environment variable: /home/conan/.conan/data/pcre2/10.36/_/_/package/c5cd7a5e52d42ff90834b8d0393b4067fc309680/bin
automake/1.16.3: Appending PATH env var with : /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin
automake/1.16.3: Setting ACLOCAL to /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/aclocal
automake/1.16.3: Setting AUTOMAKE_DATADIR to /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/share
automake/1.16.3: Setting AUTOMAKE_LIBDIR to /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/share/automake-1.16
automake/1.16.3: Setting AUTOMAKE_PERLLIBDIR to /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/share/automake-1.16
automake/1.16.3: Setting AUTOMAKE to /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/automake
automake/1.16.3: Append M4 include directories to AUTOMAKE_CONAN_INCLUDES environment variable
fontconfig/2.13.93: Creating FONTCONFIG_FILE environment variable: /home/conan/.conan/data/fontconfig/2.13.93/_/_/package/15af59b859610203277a1fbe9ab3cd592946e5b0/bin/etc/fonts/fonts.conf
fontconfig/2.13.93: Creating FONTCONFIG_PATH environment variable: /home/conan/.conan/data/fontconfig/2.13.93/_/_/package/15af59b859610203277a1fbe9ab3cd592946e5b0/bin/etc/fonts
glib/2.67.5: Appending PATH env var with: /home/conan/.conan/data/glib/2.67.5/_/_/package/95cb03f69b063acaf3e0248d172a4e0b702071f8/bin
libtool/2.4.6: Appending PATH env: /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin
libtool/2.4.6: Setting LIBTOOLIZE env to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/libtoolize
libtool/2.4.6: Setting LIBTOOL_PREFIX environment variable to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4
libtool/2.4.6: Setting LIBTOOL_DATADIR environment variable to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share
libtool/2.4.6: Setting LIBTOOL_PKGAUXDIR environment variable to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share/libtool/build-aux
libtool/2.4.6: Setting LIBTOOL_PKGLTDLDIR environment variable to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share/libtool
libtool/2.4.6: Setting LIBTOOL_ACLOCALDIR environment variable to /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share/aclocal
libtool/2.4.6: Appending ACLOCAL_PATH env: /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share/aclocal
libtool/2.4.6: Appending AUTOMAKE_CONAN_INCLUDES environment variable: /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/share/aclocal
cairo/1.17.4: Applying build-requirement: libtool/2.4.6
cairo/1.17.4: Applying build-requirement: pkgconf/1.7.3
cairo/1.17.4: Applying build-requirement: automake/1.16.3
cairo/1.17.4: Applying build-requirement: autoconf/2.69
cairo/1.17.4: Applying build-requirement: m4/1.4.18
Downloading conan_sources.tgz
cairo/1.17.4: Configuring sources in /home/conan/.conan/data/cairo/1.17.4/_/_/source

cairo/1.17.4: Copying sources to build folder
cairo/1.17.4: Building your package in /home/conan/.conan/data/cairo/1.17.4/_/_/build/047d690cb0db4aa8f5feeff56c57a66b01987249
cairo/1.17.4: Generator pkg_config created fontconfig.pc
cairo/1.17.4: Generator pkg_config created gresource.pc
cairo/1.17.4: Generator pkg_config created glib-2.0.pc
cairo/1.17.4: Generator pkg_config created gobject-2.0.pc
cairo/1.17.4: Generator pkg_config created gmodule-no-export-2.0.pc
cairo/1.17.4: Generator pkg_config created gmodule-2.0.pc
cairo/1.17.4: Generator pkg_config created gio-2.0.pc
cairo/1.17.4: Generator pkg_config created gio-unix-2.0.pc
cairo/1.17.4: Generator pkg_config created gthread-2.0.pc
cairo/1.17.4: Generator pkg_config created gmodule-export-2.0.pc
cairo/1.17.4: Generator pkg_config created glib.pc
cairo/1.17.4: Generator pkg_config created pixman-1.pc
cairo/1.17.4: Generator pkg_config created freetype2.pc
cairo/1.17.4: Generator pkg_config created expat.pc
cairo/1.17.4: Generator pkg_config created libuuid.pc
cairo/1.17.4: Generator pkg_config created libffi.pc
cairo/1.17.4: Generator pkg_config created libpcre32.pc
cairo/1.17.4: Generator pkg_config created libpcre16.pc
cairo/1.17.4: Generator pkg_config created libpcre.pc
cairo/1.17.4: Generator pkg_config created libpcreposix.pc
cairo/1.17.4: Generator pkg_config created pcre.pc
cairo/1.17.4: Generator pkg_config created libelf.pc
cairo/1.17.4: Generator pkg_config created mount.pc
cairo/1.17.4: Generator pkg_config created libsepol.pc
cairo/1.17.4: Generator pkg_config created libselinux.pc
cairo/1.17.4: Generator pkg_config created libpng.pc
cairo/1.17.4: Generator pkg_config created libbrotlicommon.pc
cairo/1.17.4: Generator pkg_config created libbrotlienc.pc
cairo/1.17.4: Generator pkg_config created libbrotlidec.pc
cairo/1.17.4: Generator pkg_config created brotli.pc
cairo/1.17.4: Generator pkg_config created libpcre2-32.pc
cairo/1.17.4: Generator pkg_config created libpcre2-16.pc
cairo/1.17.4: Generator pkg_config created libpcre2-8.pc
cairo/1.17.4: Generator pkg_config created libpcre2-posix.pc
cairo/1.17.4: Generator pkg_config created libpcre2.pc
cairo/1.17.4: Generator pkg_config created zlib.pc
cairo/1.17.4: Generator pkg_config created bzip2.pc
cairo/1.17.4: Calling build()
*** No GTK-Doc found, documentation won't be generated ***
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/aclocal -I build ${ACLOCAL_FLAGS}
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
autoreconf: configure.ac: tracing
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
autoreconf: running: /home/conan/.conan/data/libtool/2.4.6/_/_/package/e33671439e0af363817f093b37f44bf2e210d7b4/bin/libtoolize --copy
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
autoreconf: running: /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autoconf
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
autoreconf: running: /home/conan/.conan/data/autoconf/2.69/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/autoheader
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
autoreconf: running: /home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/automake --add-missing --copy --no-force
build/configure.ac.system:114: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2590: _AC_COMPILE_IFELSE is expanded from...
/home/conan/w/BuildSingleReference/.conan/data/autoconf/2.69/_/_/build/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/source_subfolder/lib/autoconf/general.m4:2606: AC_COMPILE_IFELSE is expanded from...
build/configure.ac.system:114: the top level
build/Makefile.am.changelog:57: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:105:   'build/Makefile.am.changelog' included from here
perf/Makefile.sources:7: warning: source file '../src/cairo-time.c' is in a subdirectory,
perf/Makefile.sources:7: but option 'subdir-objects' is disabled
perf/Makefile.am:3:   'perf/Makefile.sources' included from here
automake: warning: possible forward-incompatibility.
automake: At least a source file is in a subdirectory, but the 'subdir-objects'
automake: automake option hasn't been enabled.  For now, the corresponding output
automake: object file(s) will be placed in the top-level directory.  However,
automake: this behaviour will change in future Automake versions: they will
automake: unconditionally cause object files to be placed in the same subdirectory
automake: of the corresponding sources.
automake: You are advised to start using 'subdir-objects' option throughout your
automake: project, to avoid future incompatibilities.
perf/Makefile.sources:15: warning: source file '../src/cairo-error.c' is in a subdirectory,
perf/Makefile.sources:15: but option 'subdir-objects' is disabled
perf/Makefile.am:3:   'perf/Makefile.sources' included from here
perf/Makefile.sources:18: warning: source file '../src/cairo-error.c' is in a subdirectory,
perf/Makefile.sources:18: but option 'subdir-objects' is disabled
perf/Makefile.am:3:   'perf/Makefile.sources' included from here
perf/Makefile.sources:18: warning: source file '../src/cairo-hash.c' is in a subdirectory,
perf/Makefile.sources:18: but option 'subdir-objects' is disabled
perf/Makefile.am:3:   'perf/Makefile.sources' included from here
test/Makefile.am:354: warning: user target 'recheck' defined here ...
/home/conan/.conan/data/automake/1.16.3/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/share/automake-1.16/am/check.am: ... overrides Automake target 'recheck' defined here
autoreconf: Leaving directory `.'
cairo/1.17.4: Calling:
 > ./configure '--datarootdir=/home/conan/.conan/data/cairo/1.17.4/_/_/package/047d690cb0db4aa8f5feeff56c57a66b01987249/res' '--enable-ft=yes' '--enable-gobject=yes' '--enable-fc=yes' '--enable-xlib=no' '--enable-xlib_xrender=no' '--enable-xcb=no' '--enable-shared=no' '--enable-static=yes' '--prefix=/home/conan/.conan/data/cairo/1.17.4/_/_/package/047d690cb0db4aa8f5feeff56c57a66b01987249' '--bindir=${prefix}/bin' '--sbindir=${prefix}/bin' '--libexecdir=${prefix}/bin' '--libdir=${prefix}/lib' '--includedir=${prefix}/include' '--oldincludedir=${prefix}/include' 
checking for gcc... clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether clang accepts -g... yes
checking for clang option to accept ISO C89... none needed
checking whether clang understands -c and -o together... yes
checking how to run the C preprocessor... clang -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for unistd.h... (cached) yes
checking sys/ioctl.h usability... yes
checking sys/ioctl.h presence... yes
checking for sys/ioctl.h... yes
checking for typeof syntax and keyword spelling... typeof
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports the include directive... yes (GNU style)
checking whether make supports nested variables... yes
checking dependency style of clang... gcc3
checking whether make supports nested variables... (cached) yes
checking for ar... ar
checking the archiver (ar) interface... ar
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for fgrep... /bin/grep -F
checking for ld used by clang... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from clang object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if clang supports -fno-rtti -fno-exceptions... yes
checking for clang option to produce PIC... -fPIC -DPIC
checking if clang PIC flag -fPIC -DPIC works... yes
checking if clang static flag -static works... yes
checking if clang supports -c -o file.o... yes
checking if clang supports -c -o file.o... (cached) yes
checking whether the clang linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking pkg-config is at least version 0.9.0... yes
checking whether to build gtk-doc documentation... no
checking for gtkdoc-check... no
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for find... /usr/bin/find
checking for xargs... /usr/bin/xargs
checking for gcc... (cached) clang
checking whether we are using the GNU C compiler... (cached) yes
checking whether clang accepts -g... (cached) yes
checking for clang option to accept ISO C89... (cached) none needed
checking whether clang understands -c and -o together... (cached) yes
checking how to run the C preprocessor... clang -E
checking whether we are using the GNU C++ compiler... yes
checking whether clang++ accepts -g... yes
checking dependency style of clang++... gcc3
checking how to run the C++ preprocessor... clang++ -E
checking for ld used by clang++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the clang++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for clang++ option to produce PIC... -fPIC -DPIC
checking if clang++ PIC flag -fPIC -DPIC works... yes
checking if clang++ static flag -static works... yes
checking if clang++ supports -c -o file.o... yes
checking if clang++ supports -c -o file.o... (cached) yes
checking whether the clang++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for inline... inline
checking pkg-config is at least version 0.9.0... yes
checking for supported warning flags... 
checking whether clang supports -Werror... yes
checking whether clang supports -errwarn... no
checking whether clang supports -Wall... yes
checking whether clang supports -Wextra... yes
checking whether clang supports -Wmissing-declarations... yes
checking whether clang supports -Werror-implicit-function-declaration... yes
checking whether clang supports -Wpointer-arith... yes
checking whether clang supports -Wwrite-strings... yes
checking whether clang supports -Wsign-compare... yes
checking whether clang supports -Wpacked... yes
checking whether clang supports -Wswitch-enum... yes
checking whether clang supports -Wmissing-format-attribute... yes
checking whether clang supports -Wvolatile-register-var... yes
checking whether clang supports -Wstrict-aliasing=2... yes
checking whether clang supports -Winit-self... yes
checking whether clang supports -Wunsafe-loop-optimizations... no
checking whether clang supports -Wno-missing-field-initializers... yes
checking whether clang supports -Wno-unused-parameter... yes
checking whether clang supports -Wno-attributes... yes
checking whether clang supports -Wno-long-long... yes
checking whether clang supports -Winline... yes
checking whether clang supports -erroff=E_ENUM_TYPE_MISMATCH_ARG... no
checking whether clang supports -erroff=E_ENUM_TYPE_MISMATCH_OP... no
checking whether clang supports -fno-strict-aliasing... yes
checking whether clang supports -fno-common... yes
checking whether clang supports -Wp,-D_FORTIFY_SOURCE=2... yes
checking whether clang supports -Wunused-but-set-variable -Wno-unused-but-set-variable... no
checking which warning flags were supported...  -Wall -Wextra -Wmissing-declarations -Werror-implicit-function-declaration -Wpointer-arith -Wwrite-strings -Wsign-compare -Wpacked -Wswitch-enum -Wmissing-format-attribute -Wvolatile-register-var -Wstrict-aliasing=2 -Winit-self -Wno-missing-field-initializers -Wno-unused-parameter -Wno-attributes -Wno-long-long -Winline -fno-strict-aliasing -fno-common -Wp,-D_FORTIFY_SOURCE=2
checking how to enable unused result warnings... __attribute__((__warn_unused_result__))
checking how to allow undefined symbols in shared libraries used by test suite... -Wl,--allow-shlib-undefined
checking whether byte ordering is bigendian... no
checking whether float word ordering is bigendian... checking for strings... strings
no
checking for native atomic primitives... cxx11
checking whether atomic ops require a memory barrier... no
checking size of void *... 8
checking size of int... 4
checking size of long... 8
checking size of long long... 8
checking size of size_t... 8
checking for native Win32... no
checking for Sun Solaris (non-POSIX ctime_r)... no
checking for cos in -lm... yes
checking for sched_yield in -lrt... yes
checking for shm_open in -lrt... yes
checking for connect in -lsocket... no
checking for __builtin_return_address(0)... yes
checking for stdint.h... (cached) yes
checking for inttypes.h... (cached) yes
checking sys/int_types.h usability... no
checking sys/int_types.h presence... no
checking for sys/int_types.h... no
checking for uint64_t... yes
checking for uint128_t... no
checking for __uint128_t... yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking for unistd.h... (cached) yes
checking signal.h usability... yes
checking signal.h presence... yes
checking for signal.h... yes
checking poll.h usability... yes
checking poll.h presence... yes
checking for poll.h... yes
checking for sys/stat.h... (cached) yes
checking sys/socket.h usability... yes
checking sys/socket.h presence... yes
checking for sys/socket.h... yes
checking sys/poll.h usability... yes
checking sys/poll.h presence... yes
checking for sys/poll.h... yes
checking sys/un.h usability... yes
checking sys/un.h presence... yes
checking for sys/un.h... yes
checking for alarm... yes
checking sched.h usability... yes
checking sched.h presence... yes
checking for sched.h... yes
checking for sched_getaffinity... yes
checking sys/mman.h usability... yes
checking sys/mman.h presence... yes
checking for sys/mman.h... yes
checking for mmap... yes
checking time.h usability... yes
checking time.h presence... yes
checking for time.h... yes
checking for clock_gettime... yes
checking fenv.h usability... yes
checking fenv.h presence... yes
checking for fenv.h... yes
checking for feenableexcept... yes
checking for fedisableexcept... yes
checking for feclearexcept... yes
checking libgen.h usability... yes
checking libgen.h presence... yes
checking for libgen.h... yes
checking byteswap.h usability... yes
checking byteswap.h presence... yes
checking for byteswap.h... yes
checking for signal.h... (cached) yes
checking setjmp.h usability... yes
checking setjmp.h presence... yes
checking for setjmp.h... yes
checking for fenv.h... (cached) yes
checking sys/wait.h usability... yes
checking sys/wait.h presence... yes
checking for sys/wait.h... yes
checking for ctime_r... yes
checking for localtime_r... yes
checking for gmtime_r... yes
checking for drand48... yes
checking for flockfile... yes
checking for funlockfile... yes
checking for getline... yes
checking for link... yes
checking for strndup... yes
checking for sys/stat.h... (cached) yes
checking io.h usability... no
checking io.h presence... no
checking for io.h... no
checking for mkdir... yes
checking mkdir variant... mkdir(path, mode)
checking for fork... yes
checking for waitpid... yes
checking for raise... yes
checking for VALGRIND... no
no
checking for compress in -lz... yes
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking for lzo2a_decompress in -llzo2... no
checking for dlsym in -ldl... yes
checking for dlfcn.h... (cached) yes
checking xlocale.h usability... yes
checking xlocale.h presence... yes
checking for xlocale.h... yes
checking for newlocale... yes
checking for strtod_l... yes
checking for X... no
checking for cairo's XCB/SHM functions feature... 
checking whether cairo's XCB/SHM functions feature could be enabled... no (requires --enable-xcb)
checking for cairo's Quartz surface backend feature... 
checking ApplicationServices/ApplicationServices.h usability... no
checking ApplicationServices/ApplicationServices.h presence... no
checking for ApplicationServices/ApplicationServices.h... no
checking CoreGraphics/CoreGraphics.h usability... no
checking CoreGraphics/CoreGraphics.h presence... no
checking for CoreGraphics/CoreGraphics.h... no
checking whether cairo's Quartz surface backend feature could be enabled... no (requires CoreGraphics framework)
checking for cairo's Quartz font backend feature... 
checking whether cairo's Quartz font backend feature could be enabled... no (requires CoreGraphics framework)
checking for cairo's Microsoft Windows surface backend feature... 
checking whether cairo's Microsoft Windows surface backend feature could be enabled... no (requires a Win32 platform)
checking for cairo's Microsoft Windows font backend feature... 
checking whether cairo's Microsoft Windows font backend feature could be enabled... no (requires a Win32 platform)
checking for cairo's PNG functions feature... 
checking for png... yes
checking whether cairo's PNG functions feature could be enabled... yes
checking for cairo's EGL functions feature... 
checking whether cairo's EGL functions feature could be enabled... no (not required by any backend)
checking for cairo's GLX functions feature... 
checking whether cairo's GLX functions feature could be enabled... no (not required by any backend)
checking for cairo's WGL functions feature... 
checking whether cairo's WGL functions feature could be enabled... no (not required by any backend)
checking for cairo's script surface backend feature... 
checking whether cairo's script surface backend feature could be enabled... yes
checking for cairo's FreeType font backend feature... 
checking for FREETYPE... no
checking for freetype-config... no
checking whether cairo's FreeType font backend feature could be enabled... no (freetype-config not found in path or $FREETYPE_CONFIG)
configure: error:  FreeType font backend feature could not be enabled
cairo/1.17.4: 
WARN: Remotes registry file missing, creating default one in /home/conan/.conan/remotes.json
cairo/1.17.4: ERROR: Package '047d690cb0db4aa8f5feeff56c57a66b01987249' build failed
cairo/1.17.4: WARN: Build folder /home/conan/.conan/data/cairo/1.17.4/_/_/build/047d690cb0db4aa8f5feeff56c57a66b01987249
ERROR: cairo/1.17.4: Error in build() method, line 95
	self._build_configure()
while calling '_build_configure', line 162
	env_build = self._get_env_build()
while calling '_get_env_build', line 148
	self._env_build.configure(args=configure_args)
	ConanException: Error 1 while executing ./configure '--datarootdir=/home/conan/.conan/data/cairo/1.17.4/_/_/package/047d690cb0db4aa8f5feeff56c57a66b01987249/res' '--enable-ft=yes' '--enable-gobject=yes' '--enable-fc=yes' '--enable-xlib=no' '--enable-xlib_xrender=no' '--enable-xcb=no' '--enable-shared=no' '--enable-static=yes' '--prefix=/home/conan/.conan/data/cairo/1.17.4/_/_/package/047d690cb0db4aa8f5feeff56c57a66b01987249' '--bindir=${prefix}/bin' '--sbindir=${prefix}/bin' '--libexecdir=${prefix}/bin' '--libdir=${prefix}/lib' '--includedir=${prefix}/include' '--oldincludedir=${prefix}/include' 

```

</details>

Setting only the profile via global `-pr` flag does seem to work, as this configures and compiles fine:

```
docker run --rm conanio/clang11 bash -c "conan install cairo/1.17.4@ --build missing -pr ~/.conan/profiles/default -o cairo:with_xcb=False -o cairo:with_xlib=False -o cairo:with_xlib_xrender=False -o cairo:with_glib=True"
```

**Motivation and expected behavior**
I am doing some kind of host system and build system detection and i set the corresponding conan profiles via `-pr:h` and `-pr:b`. In the case, where i have the same host and build system, i get this strange error. For me it seems like some conan bug, since my expectaion would be that providing the same profile for host and build system should behave exactly the same as providing this profile only with `-pr` flag. Is my assumption wrong here?

--- a/recipes/cairo/all/conanfile.py
+++ b/recipes/cairo/all/conanfile.py
@@ -66,7 +66,7 @@ class CairoConan(ConanFile):
             if self.options.with_xlib or self.options.with_xlib_xrender or self.options.with_xcb:
                 self.requires("xorg/system")
         if self.options.get_safe("with_glib", True):
-            self.requires("glib/2.67.5")
+            self.requires("glib/2.68.0")
         self.requires("zlib/1.2.11")
         self.requires("pixman/0.40.0")
         self.requires("libpng/1.6.37")

--- a/recipes/cairo/all/conanfile.py
+++ b/recipes/cairo/all/conanfile.py
@@ -83,8 +83,8 @@ class CairoConan(ConanFile):
         return self.settings.compiler == 'Visual Studio'
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        os.rename('cairo-%s' % self.version, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
 
     def build(self):
         for patch in self.conan_data["patches"][self.version]:
@@ -145,7 +145,8 @@ class CairoConan(ConanFile):
         if self.settings.compiler in ["gcc", "clang", "apple-clang"]:
             self._env_build.flags.append("-Wno-enum-conversion")
 
-        self._env_build.configure(args=configure_args)
+        with tools.run_environment(self):
+            self._env_build.configure(args=configure_args)
         return self._env_build
 
     def _build_configure(self):

--- a/recipes/cairo/all/conanfile.py
+++ b/recipes/cairo/all/conanfile.py
@@ -66,17 +66,17 @@ class CairoConan(ConanFile):
             if self.options.with_xlib or self.options.with_xlib_xrender or self.options.with_xcb:
                 self.requires("xorg/system")
         if self.options.get_safe("with_glib", True):
-            self.requires("glib/2.68.0")
+            self.requires("glib/2.68.1")
         self.requires("zlib/1.2.11")
         self.requires("pixman/0.40.0")
         self.requires("libpng/1.6.37")
 
     def build_requirements(self):
         if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires('msys2/20200517')
+            self.build_requires('msys2/cci.latest')
         if not self._is_msvc:
             self.build_requires("libtool/2.4.6")
-            self.build_requires("pkgconf/1.7.3")
+            self.build_requires("pkgconf/1.7.4")
 
     @property
     def _is_msvc(self):

--- a/recipes/cairo/all/conanfile.py
+++ b/recipes/cairo/all/conanfile.py
@@ -51,6 +51,8 @@ class CairoConan(ConanFile):
             del self.options.with_xcb
 
     def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
         del self.settings.compiler.cppstd
         del self.settings.compiler.libcxx
         if self._is_msvc:

